README.md

```diff
@@ -1,62 +1,47 @@
-It needs only two settings to run.
-The *Main class* should be set to **ch.wisv.areafiftylan.Application** and the *Use classpath of module* should be set to your project module.
-Finally, enable the dev profile by entering `dev` in the `Active Profiles` box.
-
-That's it! Select your new Spring Boot configuration and click Run!. This should launch the application on localhost:9000
-
-#### Running with Gradle
-It is also possible to run the application directly from the command line.
-This is done by running the command `./gradlew bootRunDev`, the application will now start just as usual.
-
-## Deploying
+# LANcie API
+This is the repository for the LANcie-API. The LANcie-API handles everything from registering users to buying tickets and reserving a seat. You can use your own front-end to interact with the API, but you can also use the [LANcie frontend](https://github.com/AreaFiftyLAN/lancie-frontend).
+
+## lancie-api
+The API is a Spring based application to suit the needs of a LAN-party.
+
+### Tools
+-   [PostgresQL](https://www.postgresql.org/)
+-   [mailcatcher](https://mailcatcher.me/), mailcatcher creates a mailserver locally on your pc. All mail sent from the API is cought here, you end up with a mailbox with every outgoing mailaddress. Unix-like systems: `gem install mailcatcher`. Windows users can try mailcatcher as well, but [Papercut](https://github.com/changemakerstudios/papercut) has an easier installation. 
+
+### Run
+1.  Import the project into IntelliJ IDEA, we really recommend using [IntelliJ IDEA Ultimate Edition](https://www.jetbrains.com/idea/), since it includes all the support for Spring. You could use another IDE, but we do not recommend this
+2.  Make sure you have installed the `Lombok Plugin`
+2.  Enable annotation processing, this can be enabled in `Settings > Build, Execution, Deployment > Compiler > Annotation Processors`. Here you have to check the checkmark that says `Enable Annotation Processors`
+3.  Copy `config/application.properties.sample` to `config/application.properties`. The sample properties assume a working PostgreSQL installation running in the background.
+
+  You should change:
+  - `spring.datasource.[…]` (`url`, `username`, `password`) to your database url and credentials
+  - `flyway.enabled` to `false`
+  - `a5l.molliekey` and `a5l.googleMapsAPIkey` to their respective keys if you have those
+  - all occurrences of `areafiftylan.nl` to `localhost:5100`
+ 
+  You should comment out:
+  - `spring.jpa.properties.hibernate.hbm2ddl.auto`
+  - `logging.config`
+  
+  You should comment in:
+  - `spring.jpa.hibernate.ddl-auto`
+
+4.  Go to the Run/Debug Configuration window `Run -> Edit Configurations`, and add a new Spring Boot configuration. It needs only two settings to run. The `Main class` should be set to `ch.wisv.areafiftylan.Application` and the `Use classpath of module` should be set to your project module. Finally, enable the dev profile by entering `dev` in the `Active Profiles` box.
```

Comment:
You could do this even faster by just right clicking the `Application` file and clicking **Run**. The only thing left to do is filling in the `dev` profile.